### ISSUE TITLE
Bug: version 3.0.0 doesn't start, with a exec format error when using assimilate within docker

### ISSUE THREAD

**Issue by fungiboletus** [jart/cosmopolitan#1338]
### What happened?

I have a `Dockerfile` to run redbean and using `--assimilate`, it works well with version 2.2 but version 3.0.0 results in some `exec format error`.

Works:
```dockerfile
FROM alpine:latest
RUN apk add curl
RUN curl -o /redbean-2.2.com https://redbean.dev/redbean-2.2.com
RUN sh /redbean-2.2.com --assimilate
RUN chmod +x /redbean-2.2.com
CMD ["/redbean-2.2.com", "-h"]
```

Doesn't work:
```dockerfile
FROM alpine:latest
RUN apk add curl
RUN curl -o /redbean-3.0.0.com https://redbean.dev/redbean-3.0.0.com
RUN sh /redbean-3.0.0.com --assimilate
RUN chmod +x /redbean-3.0.0.com
CMD ["/redbean-3.0.0.com", "-h"]
```

### Version

redbean 3.0.0

### What operating system are you seeing the problem on?

Linux

### Relevant log output

```shell
exec /app/redbean.com: exec format error
```

**Comment by jart** [comment]
We removed the assimilate flag in favor of having it be a separate program: https://cosmo.zip/pub/cosmos/bin/. You don't have to assimilate your ape binaries though. Try the `ape` interpreter instead. See the gotchas published at https://redbean.dev/#linux and https://github.com/Mozilla-Ocho/llamafile/?tab=readme-ov-file#linux. You could setup your dockerfile to use this interpreter, because docker doesn't support `binfmt_misc`. For example, It'd probably look like this:

```dockerfile
FROM alpine:latest
RUN apk add curl
RUN curl -o /ape https://cosmo.zip/pub/cosmos/bin/ape-x86_64.elf
RUN chmod +x /ape
RUN curl -o /redbean https://redbean.dev/redbean-3.0.0.com
RUN chmod +x /redbean-3.0.0.com
CMD ["/ape", "/redbean", "-h"]
```

**Comment by jart** [comment]
Please confirm the docker code above works for you, and I'll add it to the readme.

**Comment by fungiboletus** [comment]
Thanks for the help! I tried with the new `ape` interpreter and it works.

The `--assimilate` CLI flag didn't fail, so I didn't notice that it wasn't working anymore.

I noticed that using `ape-x86_64.elf` on the `linux/amd64` docker platform, on a arm64 VZ VM with rosetta didn't work. But it works on native hardware so it's probably related to the emulation. `ape-arm64.elf` worked on the `linux/arm64` docker platform on the same VM, without emulation.

I used this code to share the same dockerfile between the two platforms:
```dockerfile
RUN if [ "${TARGETARCH}" = "amd64" ]; then \
        curl --resolve cosmo.zip:443:34.136.86.162 -o /ape https://cosmo.zip/pub/cosmos/bin/ape-x86_64.elf; \
    elif [ "${TARGETARCH}" = "arm64" ]; then \
        curl --resolve cosmo.zip:443:34.136.86.162 -o /ape https://cosmo.zip/pub/cosmos/bin/ape-arm64.elf; \
    else \
        echo "Unsupported architecture: ${TARGETARCH}"; exit 1; \
    fi
```

By the way, I also had to have a manual resolve of `cosmo.zip` because the DNS lookup returns nothing on the the builder nodes I use. It resolves on my machine. `redbean.dev` resolves on the builder nodes, so I guess `cosmo.zip` is still in a disallow list somewhere.
